app/assets/javascripts/merge_status.coffee

```diff
@@ -42,6 +42,13 @@ class MergeStatusPoller
         container.innerHTML = html
         @onPageChange()
 
+  reloadUrl: =>
+    rollback_checkbox = document.querySelector('input[name="rollbackable"]:checked')
```

Comment:
So as you figured, this part of the UI really wasn't meant to hold state.

That being said, if you want to do this, a cleaner way could be to serialize the form in a more generic way, e.g. get the form element and iterate over the inputs and build the query string.